test/testUtility.js

```diff
@@ -0,0 +1,177 @@
+const config = require('config');
+const jwt = require('jsonwebtoken');
+const path = require('path');
+const mongoRestore = require('mongodb-restore');
+const { ObjectID } = require('bson');
+const { assert } = require('chai');
+
+const assertThrowsAsync = async (fn, errorMessage) => {
+  let f = () => {
+    // empty function
+  };
+  try {
+    await fn();
+  } catch (e) {
+    f = () => {
+      throw e;
+    };
+  } finally {
+    if (errorMessage) {
+      assert.throw(f, errorMessage);
+    } else {
+      assert.throw(f);
+    }
+  }
+};
```

Comment:
chai has a plugin to use assertion for promises 
https://www.chaijs.com/plugins/chai-as-promised/
Let's use it instead of manually checking async function result

Reply:
> chai has a plugin to use assertion for promises
> https://www.chaijs.com/plugins/chai-as-promised/
> Let's use it instead of manually checking async function result

Thanks @amin , I think it's for another purpose , it's for testing promises in non-async functions, but I'll test it , 
but our problem here is that the function doesn't return anything,  I'll check to see if chai-as-promised solve the problem or not.

Reply:
@aminlatifi 
This plugin doesn't solve the problem that we have to return promise in our functions that want to be tested, 
this plugin is just for not using  promise in test codes,and I think after `async-await` coming this library (and mocha support async functions) it's useless. you can check that their last commit is for 3 years ago,
Please take a look at this  topic :
https://stackoverflow.com/questions/26571328/how-do-i-properly-test-promises-with-mocha-and-chai

PS : I think if we want to test some functions , those functions should've output then we can test them otherwise we should find a magical solution

Reply:
OK, let's close this.